packages/authorization-process/src/services/authorizationService.ts

```diff
@@ -336,6 +346,44 @@ export function authorizationServiceBuilder(
         showUsers: isClientConsumer(client.data.consumerId, organizationId),
       };
     },
+    async addUser(
+      {
+        clientId,
+        userId,
+        authData,
+      }: {
+        clientId: ClientId;
+        userId: UserId;
+        authData: AuthData;
+      },
+      correlationId: string,
+      logger: Logger
+    ): Promise<{ client: Client; showUsers: boolean }> {
+      logger.info(`Binding client ${clientId} with user ${userId}`);
+      const client = await retrieveClient(clientId, readModelService);
+      assertOrganizationIsClientConsumer(authData.organizationId, client.data);
+      await assertSecurityUser(authData.selfcareId, authData.userId, userId);
+      if (client.data.users.includes(userId)) {
+        throw userAlreadyAssigned(clientId, userId);
+      }
+      const updatedClient: Client = {
+        ...client.data,
+        users: [...client.data.users, userId],
+      };
+
+      await repository.createEvent(
+        toCreateEventClientUserAdded(
+          userId,
+          updatedClient,
+          client.metadata.version,
+          correlationId
+        )
+      );
+      return {
+        client: updatedClient,
+        showUsers: updatedClient.consumerId === authData.organizationId,
```

Comment:
This check could be removed in favor of an harcoded `true` since it is already checkd by the `assertOrganizationIsClientConsumer`